doc/htmldoc/hpc/mpi_processes.rst

```diff
@@ -0,0 +1,84 @@
+.. _mpi_process:
+
+MPI process
+===========
+
+
+An MPI process is a set of data and instructions with its own memory that must be explictly shared between processes (also known as CPUs).
```

Comment:
If the intention with this sentence is to just introduce the whole MPI process section I would suggest to be a little bit more clear like:
"An MPI process is one of the instances of a computer program that is being executed by one or many threads and employs a Message Passing Interface (MPI) to communicate its data with the other instances."

Otherwise, if the intention with the sentence was to be more didactic for users with no knowledge of "MPI processes" I would expand a little bit on what is a process and what MPI protocols are and how these come together.